libp2p/pubsub/gossipsub.py

```diff
@@ -442,29 +508,36 @@ def _get_in_topic_gossipsub_peers_from_minus(
         from_id_str = sender_peer_id
 
         # Get list of all seen (seqnos, from) from the (seqno, from) tuples in seen_messages cache
-        seen_seqnos_and_peers = [seqno_and_from
-                                 for seqno_and_from in self.pubsub.seen_messages.keys()]
+        seen_seqnos_and_peers = [
+            seqno_and_from
+            for seqno_and_from in self.pubsub.seen_messages.keys()
+        ]
 
         # Add all unknown message ids (ids that appear in ihave_msg but not in seen_seqnos) to list
         # of messages we want to request
-        msg_ids_wanted = [msg_id for msg_id in ihave_msg.messageIDs
-                          if literal_eval(msg_id) not in seen_seqnos_and_peers]
+        # FIXME: Update type of message ID
+        msg_ids_wanted: List[Any] = [
+            msg_id
+            for msg_id in ihave_msg.messageIDs
+            if literal_eval(msg_id) not in seen_seqnos_and_peers
+        ]
 
         # Request messages with IWANT message
         if msg_ids_wanted:
             await self.emit_iwant(msg_ids_wanted, from_id_str)
 
-    async def handle_iwant(self, iwant_msg, sender_peer_id):
+    async def handle_iwant(self, iwant_msg: rpc_pb2.Message, sender_peer_id: str) -> None:
         """
         Forwards all request messages that are present in mcache to the requesting peer.
         """
         from_id_str = sender_peer_id
 
-        msg_ids = [literal_eval(msg) for msg in iwant_msg.messageIDs]
-        msgs_to_forward = []
+        # FIXME: Update type of message ID
+        msg_ids: List[Any] = [literal_eval(msg) for msg in iwant_msg.messageIDs]
```

Comment:
Note: Actually I'm a little bit worried about the dynamic type here. Could you add a TODO that we should investigate a better approach to parse the `msg` to `Tuple` here?